runtime/jcl/common/mgmtos.c

```diff
@@ -218,14 +218,14 @@ Java_com_ibm_lang_management_internal_SysinfoCpuTime_getCpuUtilizationImpl(JNIEn
 
 	if (NULL == tempMethod) {
 
-		tempMethod = (*env)->GetMethodID(env, clazz, "<init>", "(JJII)V");
+		tempMethod = (*env)->GetMethodID(env, clazz, "<init>", "(JJIII)V");
 		if (NULL == tempMethod) {
 			return NULL;
 		}
 
 		JCL_CACHE_SET(env, MID_com_ibm_lang_management_SysinfoCpuTime_getCpuUtilization_init, tempMethod);
 	}
-
+printf("Found tempMethod - getting time now\n");fflush(stdout);
```

Comment:
There's a printf here and in other places that needs to be removed.

Reply:
Fix applied in ffdf65f8be493319c906b24c60b752782364cb35